next/content/docs/manage/branches.md

```diff
@@ -9,11 +9,11 @@ redirectFrom:
 Data resides in a branch. Each Neon project has a root branch called `main`. You can create child branches from `main` or from previously created branches. A branch can contain multiple databases and users. Tier limits define the number of branches you can create in a project and the amount of data you can store in a branch.
 
 A child branch is a copy-on-write clone the data in the parent branch. You can modify the data in a branch without affecting the data in the parent branch.
-For more information about branches and how you can use them in your development workflows, see [Branching](../../conceptual-guides/branching).
+For more information about branches and how you can use them in your development workflows, see [Branching](/docs/introduction/branching).
 
 You can create and manage branches using the Neon Console or [Neon API](https://neon.tech/api-reference/v2/). This topic covers both methods.
 
-Before you can create a branch, you must have a Neon project. If you do not have a Neon project, see [Create a project](../projects/#create-a-project).
+Before you can create a branch, you must have a Neon project. If you do not have a Neon project, see [Create a project](/docs/projects/#create-a-project).
```

Comment:
This link gives a 404. Probably intended to be `/docs/manage/projects#create-a-project`?

Reply:
> These just redirect to the page without a trailing slash. Unless there's some other reason I'm not aware of, maybe these should be changed?

Fixed these in a new PR. Thanks for point that out.